src/main/java/com/github/timm/cucumber/generate/GenerateRunnersMojo.java

```diff
@@ -190,13 +190,20 @@
     public void execute() throws MojoExecutionException {
 
         if (!featuresDirectory.exists()) {
-            throw new MojoExecutionException("Features directory does not exist");
+            getLog().warn("Features directory does not exist");
```

Comment:
The resources plugin just informs us about skipping the non existing resources but does include the missing directory name. The first keeps the messages standardized, the latter is helpful.

```
[INFO] --- maven-resources-plugin:2.6:testResources (default-testResources) @ simmetrics-example ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] skip non existing resourceDirectory /home/mpkorstanje/Projects/SimMetrics/simmetrics/simmetrics-example/src/test/resources
```
So I think this should be `skip non existing featuresDirectory /home/user/path/to/features`